step-plans/step-plans-parser/src/test/java/step/repositories/parser/DefaultDescriptionStepParserTest.java

```diff
@@ -483,14 +484,30 @@ public void testIf() throws ParsingException {
 
 		Assert.assertEquals(1,getChildren(f).size());
 	}
-	
+
 	@Test
 	public void testCheck() throws ParsingException {
 		List<AbstractStep> steps = new ArrayList<>();
-		steps.add(step("Check Expression= true \n"));
-
-		Check artefact = parseAndGetUniqueChild(steps, Check.class);
-		Assert.assertNotNull(artefact);
-		Assert.assertEquals("true",artefact.getExpression().getExpression());
+		steps.add(step("Check Expression= true \n "));
+		steps.add(step("Check Expression = key2.toString().equals('value2') \n "));

```

Comment:
Would add a test to cover the case with more than one space between Check and Expression: "Check   Expression=true"

Reply:
Added more test to cover more spaces but also non breaking space, endlines ...